lms/djangoapps/verify_student/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Student Identity Verification App
+"""
+
+default_app_config = 'lms.djangoapps.verify_student.apps.VerifyStudentConfig'
```

Comment:
Shouldn't this just be `verify_student.apps.VerifyStudentConfig`?  `lms/djangoapps` should be in the `PYTHONPATH`.